content/en/docs/contributing/docs/style-guide.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Documentation Style Guide
+linkTitle: Documentation Style Guide
+weight: 20
+description: >
+    Style Guide for Flux Documentation
+---
+## Documentation formatting standards
+
+This page gives writing style guidelines for the Flux documentation. These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+### Use upper camel case for API objects
+
+Do | Don't
+:--| :-----
+| The GitRepository resource is responsible for ... | The Git Repository resource is responsible for ... |
+
+### Use angle brackets for placeholders
+
+Use angle brackets for placeholders. Tell the reader what a placeholder represents, for example
```

Comment:
```suggestion
Use angle brackets (`<>`) for placeholders. Tell the reader what a placeholder represents, for example
```

This might make it easier for non-native speakers to instantly get what it's about.